chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/QuestionableActivityEmailJob.java

```diff
@@ -689,8 +674,28 @@ private Properties loadProperties() throws IOException {
         return props;
     }
 
-    private void setQuestionableActivityDAO(final QuestionableActivityDAO questionableActivityDao) {
-        this.questionableActivityDao = questionableActivityDao;
+    private void populateRangeDefaultsFromJobData(final JobExecutionContext context) {
```

Comment:
What if this function returned a `org.apache.commons.lang3.Range` object, or something similar? Then you could make the current "minRangeInDays and maxRangeInDays actual final static constants, and use the Range object returned from this function in the calling function to do a ".contains(range)" for the actual value